golang/internal/grpc/grpc.go

```diff
@@ -41,21 +41,25 @@ type ConnectionParams struct {
 }
 
 type (
-	DeployFunc     func(context.Context, *dogger.DeploymentLogger, *v1.DeployImageRequest, *v1.VersionData) error
-	WatchFunc      func(context.Context, string) []*common.ContainerStateItem
-	DeleteFunc     func(context.Context, string, string) error
-	SecretListFunc func(context.Context, string, string) ([]string, error)
-	SelfUpdateFunc func(context.Context, string, int32) error
-	CloseFunc      func(context.Context, agent.CloseReason) error
+	DeployFunc           func(context.Context, *dogger.DeploymentLogger, *v1.DeployImageRequest, *v1.VersionData) error
+	WatchFunc            func(context.Context, string) []*common.ContainerStateItem
+	DeleteFunc           func(context.Context, string, string) error
+	SecretListFunc       func(context.Context, string, string) ([]string, error)
+	SelfUpdateFunc       func(context.Context, string, int32) error
+	CloseFunc            func(context.Context, agent.CloseReason) error
+	ContainerCommandFunc func(context.Context, *common.ContainerCommandRequest) error
```

Comment:
no other functions have params from the proto file, at that point a peeled version of the request should be arriving independent of transport methods.
This and the following line.

Reply:
noted. made a techdept ticket for it